### PR TITLE
Add Russian localization

### PR DESCRIPTION
**Pull request by 2Grey**

### CODE REVIEW

File: Sources/KeyboardShortcuts/Localization/ru.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"record_shortcut" = "Добавить сочетание";
+"press_shortcut" = "Введите...";
```
Comment (sindresorhus) [inline Sources/KeyboardShortcuts/Localization/ru.lproj/Localizable.strings:2]: The dots are not in the English version.

Comment (2Grey) [reply Sources/KeyboardShortcuts/Localization/ru.lproj/Localizable.strings:2]: @sindresorhus, it is adaptive version of English version.
For `record_shortcut` Russian version is `Записать сочетание клавиш`;
And for `press_shortcut` Russian version is `Введите сочетание`;

For the Russian language, the translation is very long compared to the English version.

**Comment by 2Grey** [comment]
@sindresorhus, example:

[Screenshoot](https://imgur.com/FySYNkL)

File: Sources/KeyboardShortcuts/Localization/ru.lproj/Localizable.strings (hunk 1 above)
Comment (sindresorhus) [reply Sources/KeyboardShortcuts/Localization/ru.lproj/Localizable.strings]: 👍

**Pull request merged**